open-api/rest-catalog-open-api.yaml

```diff
@@ -263,7 +263,6 @@ paths:
             If not provided or empty, all top-level namespaces should be listed.
             If parent is a multipart namespace, the parts must be separated by the unit separator (`0x1F`) byte.
           required: false
-          allowEmptyValue: true
```

Comment:
Should we remove `or empty` in L263? Also should we explicitly state that `For backward compatibility, empty string is treated as absent for now`?